src/rgw/rgw_user.cc

```diff
@@ -1761,9 +1761,11 @@ int RGWUser::init(RGWUserAdminOpState& op_state)
     found = (rgw_get_user_info_by_uid(store, user_id, user_info, &op_state.objv) >= 0);
     op_state.found_by_uid = found;
   }
-  if (!user_email.empty() && !found) {
-    found = (rgw_get_user_info_by_email(store, user_email, user_info, &op_state.objv) >= 0);
-    op_state.found_by_email = found;
+  if (store->ctx()->_conf->rgw_user_unique_email) {
```

Comment:
i think the correct resolution here should be:
```diff
-  if (store->ctx()->_conf.get_val<bool>("rgw_user_unique_email")) {
+  if (store->ctx()->_conf->get_val<bool>("rgw_user_unique_email")) {
```

Reply:
Thanks